lib/pure/smtp.nim

```diff
@@ -221,20 +309,22 @@ proc auth*(smtp: AsyncSmtp, username, password: string) {.async.} =
   ## May fail with ReplyError.
 
   await smtp.sock.send("AUTH LOGIN\c\L")
-  await smtp.checkReply("334") # TODO: Check whether it's asking for the "Username:"
+  await smtp.checkReply("334") # TODO: Check whether it's asking for the 
+                               # "Username:"
```

Comment:
Why did you move part of this comment onto the next line? (and the ones below too?)